cfgov/tccp/situations.py

```diff
@@ -159,7 +159,7 @@ def get_nonconflicting_params(cls, situations):
                     "fees typically exceed the value of any rewards earned."
                 ),
                 "link": "Learn why rewards may not be as beneficial as they seem.",
-                "url": "/",
+                "url": "/about-us/newsroom/cfpb-report-highlights-consumer-frustrations-with-credit-card-rewards-programs/",
```

Comment:
The Python linter will complain about long lines like this. This (annoying formatting) works instead:

```suggestion
                "url": (
                    "/about-us/newsroom/"
                    "cfpb-report-highlights-consumer-frustrations-with-credit-card-"
                    "rewards-programs/"
                ),
```